homeworks/serhii.buhaichuk_Academik1986/3-functions/functions.js

```diff
@@ -0,0 +1,40 @@
+const OIL = 926;
+const WATER = 1000;
+const MERCURY = 1355;
+
+const weightOfOilTanker = 926;
+const weightOfWaterTanker = 1000;
+const weightOfMercuryTanker = 1355;
+
+const amountOfOilTanker = 3;
+const amountOfWaterTanker = 5;
+const amountOfMercuryTanker = 2;
```

Comment:
1. Краще уникати of у назвах
2. краще замість amount слово count. Перше в англійській означає неперелічувану кількість

```suggestion
const oilTankerWeight = 926;
const waterTankerWeight = 1000;
const mercuryTankerWeight = 1355;

const oilTankerCount = 3;
const waterTankerCount = 5;
const mercuryTankerCount = 2;
```